src/MLJInterface.jl

```diff
@@ -186,16 +188,19 @@ function fit(mlj_model::MODELS, verbosity::Int, X, y, w=AbstractFloat[])
     X = MLJModelInterface.matrix(X)
     # The FFI wrapper wants Float32 for these
     w = Float32.(w)
-    report = LightGBM.fit!(model, X, y_lgbm; verbosity=verbosity, weights=w)
+    train_metrics = LightGBM.fit!(model, X, y_lgbm; verbosity=verbosity, weights=w)
```

Comment:
This is so called because via MLJ it isn't possible (as designed so far) to get validation metrics (after all, one would use MLJ for the functionality)